service/src/main/java/bio/terra/externalcreds/ExternalCredsWebApplication.java

```diff
@@ -0,0 +1,28 @@
+package bio.terra.externalcreds;
+
+import bio.terra.common.logging.LoggingInitializer;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.ComponentScan.Filter;
+import org.springframework.context.annotation.FilterType;
+
+@SpringBootConfiguration
+@EnableAutoConfiguration
+@ComponentScan(
+    basePackages = {
+      "bio.terra.externalcreds",
+      "bio.terra.common.logging",
+      "bio.terra.common.retry.transaction"
+    },
+    excludeFilters = @Filter(type = FilterType.ANNOTATION, classes = SpringBootConfiguration.class))
+public class ExternalCredsWebApplication {
```

Comment:
the is a rename from `ExternalCredsApplication` except it no longer uses `@SpringBootApplication` because that does not give control over exclusions